### PR TITLE
WIP: Shooter velocity controls

### LINKED ISSUE #6: Velocity control
Implement velocity control subsystem

### PR DESCRIPTION
**Pull request by tsieben**

### CODE REVIEW

File: src/main/java/frc/robot/Commands/Shoot.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Systems.Shooter;
+
+public class Shoot extends Command {
+  Shooter shooter;
+  /** Creates a new Shoot. */
+  public Shoot(Shooter subsystem1) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    shooter = subsystem1;
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/Commands/Shoot.java:15]:
```suggestion
  public Shoot(Shooter shooter) {
    // Use addRequirements() here to declare subsystem dependencies.
    this.shooter = shooter;
```

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -4,37 +4,49 @@
...
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
 import frc.robot.Systems.*;
 import frc.robot.Systems.Chassis.*;
 import frc.robot.Commands.*;
+import frc.robot.Constants.Constants;
 
 public class RobotContainer {
-  private Chassis m_chassis;
+  //private Chassis m_chassis;
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/RobotContainer.java:20]: Why comment out the drive code? I know it's not used for the shooter test, but shouldn't interfere with it

File: src/main/java/frc/robot/Systems/Shooter.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  private RelativeEncoder shooterTopENC;
+  private RelativeEncoder shooterBottomENC;
+
+  /** Creates a new Shooter. */
+  public Shooter() {
+    shooterTOP = new CANSparkMax(RobotConstants.FLYWHEEL_TOP_ID, MotorType.kBrushless);
+    shooterBOTTOM = new CANSparkMax(RobotConstants.FLYWHEEL_BOTTOM_ID, MotorType.kBrushless);
+
+    //shooterTOP.restoreFactoryDefaults();
+    //shooterBOTTOM.restoreFactoryDefaults();
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/Systems/Shooter.java:35]: I think I lied. This probably _is_ ok to do. I think what we don't want to do is call the `burnFlash` method, but even that isn't fully agreed upon on chief Delphi 🤷🏼

**Review by ArchdukeTim — CHANGES_REQUESTED** [review]

**Comment by siebs7317** [comment]
Closing this pull request since the decision was made to switch to Falcon/CTRE motors rather than REV/Neo.

**Pull request closed without merging**